CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Added
 - Adding the `list--edit` icon to the icons preview
+- Added the `ToggleButton` component
+- Added the `HCLSoftwareBlue 07 (20P)` color
+- Added the `selectedOpacityHover` in theme.palette.action
```

Comment:
Hello @kristineapallaHCL , nothing wrong with your additions here, also not your duty to fix

But I noticed the old additions by other contributors under Fixed > Changed are in a bit of disarray

Can we pull some of the Added stuff from there under the ###Added, like you did for yours? Thanks! @kristineapallaHCL